### PR TITLE
OCPBUGS-2219: Empty string should be valid for i18n loadType

### PR DESCRIPTION
**Pull request by jhadvig**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -109,3 +109,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.2.0 // indirect
 )
+
+replace github.com/openshift/api => github.com/jhadvig/api v0.0.0-20221018122159-ba7b5828983d
```
Comment (TheRealJon) [inline go.mod:113]: Is this meant to be merged?

**Review by TheRealJon — COMMENTED** [review]
Just the one question.

File: go.mod (hunk 1 above)
Comment (jhadvig) [reply go.mod:113]: nope ... forgot to remove :(

**Review by TheRealJon — APPROVED** [review]
/lgtm

**Pull request merged**